src/solver/linear_solver_in_place.cu

```diff
@@ -40,32 +48,32 @@ namespace micm
             for (auto j = 0; j < j_lim; ++j)
             {
               const std::size_t d_Lij_yj_first = (*d_Lij_yj).first;
-              const std::size_t d_Lij_yj_second_times_ncells = (*d_Lij_yj).second * number_of_grid_cells;
+              const std::size_t d_Lij_yj_second_times_vlens = (*d_Lij_yj).second * cuda_matrix_vector_length;
               auto d_ALU_ptr = d_ALU + d_Lij_yj_first;
-              auto d_x_ptr = d_x + d_Lij_yj_second_times_ncells;
-              d_y[tid] -= d_ALU_ptr[tid] * d_x_ptr[tid];
+              auto d_x_ptr = d_x + d_Lij_yj_second_times_vlens;
+              d_y[local_tid] -= d_ALU_ptr[local_tid] * d_x_ptr[local_tid];
```

Comment:
maybe rename to `d_Lij_yj_second_times_vector_length`?

Reply:
sure. just updated the name.